docs/faq/lukso-standard-proposal.md

```diff
@@ -0,0 +1,43 @@
+---
+title: LUKSO Standard Proposals
+description: LUKSO Frequently Asked Questions
+sidebar_position: 3
+---
+
+# Frequently Asked Questions
+
+## What are LUKSO Standard Proposals ?
+
+LUKSO Standard Proposals, often abbreviated as **LSPs**, serve as building blocks designed to create a wide range of features and possibilities within the LUKSO ecosystem and other similar blockchain networks. With these LSPs, the mission is to have standards that facilitate the use of blockchain tools that abstract the blockchain complexity, effectively minimizing the technical problems users may encounter. By abstracting this complexity, LSPs make blockchain technology as user-friendly and accessible as the familiar Web 2.0 technologies. This ease of use paves the way for broader acceptance and integration of blockchain systems in our everyday digital interactions.
+
+## Why are LSPs not proposed as ERCs?
+
+The LUKSO Standard Proposals (LSPs) are part of a larger set of core standards that are all interoperable and cohesive together. Nonetheless, they do function equally for any EVM chain, the ability to make these standards understood requires a less noisy and neutral place, than the current ERC space.
+
+Therefore, we deem a separate repo **[(LIPs/LSPs)](https://github.com/lukso-network/LIPs)** as necessary for the success and transparency of these new sets of standards, many of which use ERCs (ERC165, ERC173, ERC725) as a core basis.
+
+## What is a transaction Relayer?
+
+A transaction relayer service, often just referred to as a "relayer", is a solution designed to improve the blockchain user experience by addressing the issue of gas fees - the costs users pay when executing transactions on a blockchain network.
+
+Normally, users need to possess the blockchain's native coin to pay these gas fees. However, a relayer works around this problem. When a user want to execute a transaction, he could sign a message from their account and send it to the relayer, the relayer then will have the permission and ability to execute transactions on their behalf. Essentially, the relayer pays the gas fees for the user.
```

Comment:
not sure if the verb "possess" can be used in this sentence. It sounds a bit unusual to me.

Maybe @CallumGrindle can give some insights here? :)